resources/views/welcome.blade.php

```diff
@@ -0,0 +1,98 @@
+<!doctype html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>Laravel</title>
+
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Nunito:200,600" rel="stylesheet" type="text/css">
+
+        <!-- Styles -->
+        <style>
+            html, body {
```

Comment:
please transfer this internal css code to a externals css file

Reply:
This welcome.blade.php is temporary top page which is made by (php artisan make:auth). I'll erase this page and make my own top page in near future. So, please ignore the problem.

Reply:
better if erased now if no use.

Reply:
Okay, I got it, I'll change it and push.